foundation/foundation-mda/src/main/resources/templates/deployment/spark-operator/spark-application-base-values.yaml.vm

```diff
@@ -85,6 +85,19 @@ sparkApp:
       env:
         - name: KRAUSENING_BASE
           value: /opt/spark/krausening/base
+        #if (${enableFileStore})
+        #foreach (${fileStore} in ${fileStores})
+        - name: "${fileStore.getName()}_FS_PROVIDER"
+          value: "s3"
+        - name: "${fileStore.getName()}_FS_ACCESS_KEY_ID"
+          value: "123"
+        - name: "${fileStore.getName()}_FS_SECRET_ACCESS_KEY"
+          valueFrom:
+            secretKeyRef:
+              name: "secretKeyName"
+              key: "456"
```

Comment:
S: The secret name should match the one used in `<project>-deploy/src/main/resources/templates/sealed-secret.yaml` and the key would be more like `aws-secret-key`, not the actual secret value.

Reply:
Refactored.